### PR TITLE
Small changes in messenger-impl module classes and common-router module classes

### PR DESCRIPTION
**Pull request by AlexKazmirchuk**

### CODE REVIEW

File: softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/AbstractRequestHandler.java [hunk 1]
```diff
@@ -21,9 +21,10 @@
 	private static final Class genericClass;
 
 	static {
-		genericClass = (Class) ((ParameterizedType) AbstractRequestHandler.class
-				.getGenericSuperclass())
-				.getActualTypeArguments()[0];
+		genericClass = null;
```
Comment (Dominys) [inline softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/AbstractRequestHandler.java]: @AlexKazmirchuk why null? remove commented code

File: softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/AbstractRequestHandler.java [hunk 2]
```diff
@@ -32,7 +33,7 @@
 		request.setHeader(msg.getHeader());
 
 		// todo check if working in future
-		request.setData(mapper.convert((Map<String, Object>) msg.getData(), (Class<T>) genericClass));
+//		request.setData(mapper.convert((Map<String, Object>) msg.getData(), (Class<T>) genericClass));
```
Comment (Dominys) [inline softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/AbstractRequestHandler.java]: @AlexKazmirchuk remove commented code

File: softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/HandlerFactory.java [hunk 3]
```diff
@@ -9,7 +9,7 @@
  */
 
 @Component
-public class HandlerFactory<T extends Handler> extends AbstractFactory<T> {
+public class HandlerFactory<T extends RequestHandler> extends AbstractFactory<T> {
```
Comment (Dominys) [inline softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/HandlerFactory.java:12]: @AlexKazmirchuk Why RequestHandler? Factory can work with handlers and with routers

Comment (AlexKazmirchuk) [reply softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/HandlerFactory.java:12]: @Dominys for routers I have RouterFactory

Comment (AlexKazmirchuk) [reply softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/HandlerFactory.java:12]: factories

Comment (AlexKazmirchuk) [reply softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/HandlerFactory.java:12]: HandlerFactory and RouterFactory must have different getRouteKey() method , HandlerFactory for RequestHandlers childs, RouterFactory for CommonRouter descendants

Comment (AlexKazmirchuk) [reply softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/HandlerFactory.java:12]: AbstractFactory can work with handlers and with routers
abstract

Comment (Dominys) [reply softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/HandlerFactory.java:12]: @AlexKazmirchuk Ok.

**Pull request merged**